.github/workflows/cmake.yml

```diff
@@ -104,7 +104,7 @@ jobs:
               cc: "gcc",
               cxx: "g++",
               archiver: "7z a",
-              generators: "Ninja",
+              generators: "Ninja Multi-Config",
```

Comment:
@uwerat this is meant as a temporary change but can be kept if we e.g. want to build and delivery multi config packages which I'm a fan of and qskinny is small enough that it doesn't matter size wise